analyzere/extras/visualizations.py

```diff
@@ -0,0 +1,272 @@
+import hashlib
+
+from analyzere import LayerView
+from datetime import datetime
+from graphviz import Digraph
+from IPython.display import FileLink
+from sys import float_info
+
+
+class LayerViewVisualizer(object):
+    """Class that provides simple visualization an Analyze Re LayerViews.
+
+    Using the 'graphviz' python package, this class enables users to
+    visualize Analyze Re LayerView objects.
+
+    The class has two main visualization methods:
+
+       view()       open the rendered graph with the default application
+
+       download()   provide a download link to save the rendered graph
+
+
+       Both of these methods provide optional parameters that control the
+       formatting of the graph:
+
+           with_terms   specify if the Layer terms should be included in each
+                        node of the graph
+
+           fmt          exposes the graphvis 'format' options which include
+                        'pdf', 'png', etc.
+
+           rankdir      exposes graphvis 'rankdir' option that controls the
+                        orientation of the graph.  Options include
+                        'TB', 'LR', 'BT', 'RL', corresponding to directed
+                        graphs drawn from top to bottom, from left to right,
+                        from bottom to top, and from right to left,
+                        respectively.
+    """
+    class Sequencer(object):
```

Comment:
if you use `_counter` it will leave the method out of the generated documentation, which I believe would be desired.